packages/react-client/src/ReactFlightClient.js

```diff
@@ -573,6 +576,33 @@ function nullRefGetter() {
   }
 }
 
+function getTaskName(type: mixed): string {
+  if (type === REACT_FRAGMENT_TYPE) {
+    return '<>';
+  }
+  if (
+    typeof type === 'object' &&
+    type !== null &&
+    type.$$typeof === REACT_LAZY_TYPE
+  ) {
+    if (type._init === readChunk) {
```

Comment:
I'm thinking of maybe using `"use client"` as the task name in this case because you're about to enter the the client and likely the client code will include the name of the client component but we may not have the code loaded yet when the element is created.